packages/url_launcher/url_launcher_android/android/src/test/java/io/flutter/plugins/urllauncher/UrlLauncherTest.java

```diff
@@ -239,6 +245,68 @@ public void openWebView_opensUrlInCustomTabsWithCORSAllowedHeader() {
     assertEquals(headers.get(headerKey), passedHeaders.getString(headerKey));
   }
 
+  @Test
+  public void openWebView_opensUrlInCustomTabsWithShowTitle() {
+    Activity activity = mock(Activity.class);
+    UrlLauncher api = new UrlLauncher(ApplicationProvider.getApplicationContext());
+    api.setActivity(activity);
+    String url = "https://flutter.dev";
+    HashMap<String, String> headers = new HashMap<>();
+    String headerKey = "Content-Type";
+    headers.put(headerKey, "text/plain");
```

Comment:
Headers don't seem relevant to this test, so shouldn't need to be set up.

Reply:
Yes, you are right. Removed this